docroot/themes/humsci/humsci_basic/Gruntfile.js

```diff
@@ -107,4 +124,5 @@ module.exports = function (grunt) {
 
   grunt.registerTask('compile', ['dart-sass:dist', 'postcss:dist']);
   grunt.registerTask('ckeditor', ['dart-sass:ckeditor', 'postcss:dist']);
+  grunt.registerTask('ckeditor', ['dart-sass:preview', 'postcss:dist']);
```

Comment:
```suggestion
  grunt.registerTask('preview', ['dart-sass:preview', 'postcss:dist']);
```

Reply:
@cienvaras I changed this and pushed it up 👍

Reply:
@joegl thanks a lot! I missed this one completely.